biochemist-python/chapters/Nonlinear Curve Fitting Part 1.ipynb

```diff
@@ -0,0 +1,1436 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Inspect the data using the plot command that is available with the dataframe\n",
+    "AP_kinetics_df_transpose.plot(marker = 'o')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There is some scatter in the data, but generally, the slopes of the curves increase with increasing substrate concentration.\n",
+    "\n",
+    "Now we want to calculate the slope for each column. We can use the \"apply\" function to do this. Apply takes all of the row or column values and applies a function. There is a bit of a challenge here. We want to use the `linregress` function from scipy.stats to get the slopes. We used this function in the linear regression module and it returned five values: slope, intercept, r-value, p-value, and standard error for the best-fit line. So we need to define a new function that returns only the slope from `stats.linregress`. Here is the typical syntax for a function:\n",
```

Comment:
I removed the use of pd.apply and created the suggested for loop. I also modified the AP_kin.csv file so that it is not necessary to transpose the dataframe here or for the take home problem at the end of the module.